std/array.d

```diff
@@ -1481,7 +1481,7 @@ if (isSomeString!S)
         {
             if (inword)
             {
-                result ~= s[istart .. i];
+                result.put(s[istart .. i]);
```

Comment:
Please don't use UFCS with `put`. Given that this code is using a specific output range, it's not as bad, but in general, it's just begging for problems when using `put` with strings.

Reply:
The same goes below.